mlrun/projects/operations.py

```diff
@@ -194,6 +194,7 @@ def my_pipe(url=""):
         project = project_object or pipeline_context.project
         local = pipeline_context.is_run_local(local)
         task.metadata.labels = task.metadata.labels or labels or {}
+        project_name = project.metadata.name if project else ""
```

Comment:
what's this fix about? can you elaborate?

Reply:
removed